src/ray/common/ray_config_def.h

```diff
@@ -602,7 +602,7 @@ RAY_CONFIG(uint64_t, subscriber_timeout_ms, 300 * 1000)
 RAY_CONFIG(uint64_t, gcs_actor_table_min_duration_ms, /*  5 min */ 60 * 1000 * 5)
 
 /// Whether to enable GCS-based actor scheduling.
-RAY_CONFIG(bool, gcs_actor_scheduling_enabled, false);
+RAY_CONFIG(bool, gcs_actor_scheduling_enabled, true);
```

Comment:
Does this mean that gcs actor scheduling is enabled by default?

Reply:
It is only turned on to go through CI pipeline. Will turn it off after code review.